### DISCUSSION TITLE
Edit Toolbar disappeared

### DISCUSSION THREAD

**Discussion by joshcsmith13** [dbeaver/dbeaver#12788] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6398845/81098706-6e3e9580-8f12-11ea-9a71-c92cbb71e7b2.png)

You need to switch perspective or restart dbeaver to let changes take effect.

_Originally posted by @serge-rider in https://github.com/dbeaver/dbeaver/issues/7868#issuecomment-624210377_

This preference disappeared in the latest version of DBeaver and my File Save button disappeared from my toolbar!  :(

**Comment by joshcsmith13** [comment]
![image](https://user-images.githubusercontent.com/8902423/117050267-6571d280-acd2-11eb-8156-530c958188b8.png)

**Comment by LonwoLonwo** [comment]
Hello @joshcsmith13 
What DBeaver version and operating system do you use?

**Comment by joshcsmith13** [comment]
> What DBeaver version and operating system do you use?

DBeaver Current Version: 21.0.4.202105021514
OS:  Windows 10.0.18363

**Comment by LonwoLonwo** [comment]
Are you talking about this issue https://github.com/dbeaver/dbeaver/issues/10692 ?

**Comment by kseniiaguzeeva** [comment, marked as answer]
We removed Edit toolbar in one of the previous releases. It is not a bug.
